### DISCUSSION TITLE
Set up the Downgrade CI

### DISCUSSION THREAD

**Discussion by datejada** [TulipaEnergy/TulipaEnergyModel.jl#449] [Ideas]
Hi,
I saw this post on LinkedIn about the downgrade CI action on GitHub:

https://www.linkedin.com/posts/activity-7162427359948713984-330U?utm_source=share&utm_medium=member_desktop

@suvayu @abelsiqueira @sjvrijn, what do you think? Is it something we want to add to Tulipa?

**Comment by sjvrijn** [comment]
Sounds like a reasonable sanity-check to make sure our listed minimum dependencies are still valid, especially if we expect to support (many) older versions of packages or Julia itself.

**Comment by abelsiqueira** [comment]
I think it is a bit of an overkill in general. It explores the specific use case of package dependencies breaking and developers not noticing and then users complain. But it happens on specific cases, and it is hard to manage, he complains, because there are so many packages involved to remember everything. I recommend we don't do it now, but maybe it can be added to COPIERTemplate.jl

**Comment by clizbe** [comment]
@abelsiqueira I'm going to close this discussion - please reopen it if it's something we want to do and will forget.
